person.rb

```diff
@@ -1,18 +1,15 @@
-require_relative 'nameable'
+class Person
+  attr_accessor :name, :rentals
```

Comment:
- [ ] There should be` setters and getters for @age` variable as well for smooth running of your app. ℹ️ 
```suggestion
class Person
  attr_accessor :name, :rentals, :age
```